FAVOUR/StackOverFlow-Lite
# #159916934 Implementation of LFA feedback

**What does this PR do?**
Implement LFA feedback on UI templates.

**Description of Task to be completed?**

1.   Create a homepage and let it have only a footer.

2.   Make the sign in and sign up boxes a reasonable size on the page.

3.   Change the hover colour of the sign in and sign up button to a lighter or darker version of the base colour.

4.   Reduce the background colour intensity of the sign in and sign out  box.

5.   Reduce the size of the navbar.

6.   Reduce the font-size in the clicked-question page.

7.   Place the answer directly under the question asked.

8.   Make the mouse pointer a cursor on hovering on the button and remove the blue border that appears around the button when the clicked.

**How should this be manually tested?**
Open a terminal and run git clone https://github.com/FAVOUR/StackOverFlow-Lite.git.
run git ft-ui-templates-159916934.
open up the project in a file explorer and open index.html on a browser.

 **or**

 Check out the [gh-pages](https://favour.github.io/StackOverFlow-Lite/UI-templates/index.html)

**Any background context you want to provide?**
None.

**What are the relevant pivotal tracker stories?**
#159916934
